src/scripts/main.js

```diff
@@ -1 +1,60 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve(`First promise was resolved`);
+  });
+
+  setTimeout(() => {
+    reject(Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    resolve(`Second promise was resolved`);
+  });
+
+  document.addEventListener('contextmenu', () => {
+    resolve(`Second promise was resolved`);
+  });
+});
+
+const click = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const contextmenu = new Promise((resolve) => {
+  document.addEventListener('contextmenu', () => {
+    resolve();
+  });
+});
+
+const thirdPromise = new Promise((resolve) => {
+  Promise.all([click, contextmenu])
+    .then(() => resolve('Third promise was resolved'));
+});
+
+const success = (suc) => {
```

Comment:
```suggestion
const showSuccessMessage = (successMessage) => {
```
Better name functions according to these rules
![image](https://github.com/mate-academy/js_promises_practice_DOM/assets/96409563/463344f2-b09f-4657-8c61-4149ab133f03)